src/components/UserSearch.js

```diff
@@ -63,12 +62,18 @@ class UserSearch extends Component {
       return this.setState({ loading: false });
     }
 
-    // Limit the results to 3
-    let found = users.result.slice(0, 3);
+    // Remove already added collaborators from search results
+    let foundUsers = users.result.filter(el =>
```

Comment:
It might also be useful to check on the server that we don't add duplicate collaborators